integration-tests/tests.py

```diff
@@ -61,8 +86,12 @@ def get_run_args(mode, args, run_conf):
 for item in run_options:
     args_str = str()
     run_args = get_run_args(item, parsed_args, test_run_config)
-    for key in run_args:
-        if run_args.get(key) != None:
-            args_str += f'{key} {run_args.get(key)} '
+    args_str = make_args(run_args)
     run_tests(item, args_str)
 
+#run doubled range exist
+run_args = get_run_args(item, parsed_args, test_run_config)
+run_args['-c'] = int(run_args['-c']) * 2 + 1
+run_args['-s'] = 5001
```

Comment:
Update `run_args` inside `run_doubled_exist_test`, because `run_doubled_exist_test` currently depends on the specific value (if you change `-c` value, it fails)